@here/olp-sdk-dataservice-read/lib/client/VersionedLayerClient.ts

```diff
@@ -36,29 +36,71 @@ import {
     RequestFactory
 } from "..";
 
+export interface VersionedLayerClientParams {
+    // HRN of the catalog.
+    catalogHrn: HRN;
+    // The ID of the layer.
+    layerId: string;
+    // The [[OlpClientSettings]] instance.
+    settings: OlpClientSettings;
+    // Layer version.
+    version: number;
```

Comment:
should be optional. also please add comment about getting the last version if not set